### PR TITLE
StopForumSpam is factored out to spam-be-gone plugin

### PR DESCRIPTION
**Pull request by akhoury**
https://github.com/NodeBB/NodeBB/issues/7212

### CODE REVIEW

File: src/views/admin/manage/registration.tpl [hunk 1]
```diff
@@ -71,6 +77,11 @@
 								<div class="btn-group pull-right">
 									<button class="btn btn-success btn-xs" data-action="accept"><i class="fa fa-check"></i></button>
 									<button class="btn btn-danger btn-xs" data-action="delete"><i class="fa fa-times"></i></button>
+                                    <!-- BEGIN users.customActions -->
+                                    <button id="{users.customActions.id}" title="{users.customActions.title}" class="btn btn-xs {users.customActions.class}">
+                                        <i class="fa {users.customActions.icon}"></i>
+                                    </button>
+                                    <!-- END users.customActions -->
```
Comment (julianlam) [inline src/views/admin/manage/registration.tpl]: Looks like this might need some indentation fixes

Comment (akhoury) [reply src/views/admin/manage/registration.tpl]: yes it was sufficient

File: src/views/admin/manage/registration.tpl [hunk 2]
```diff
@@ -30,23 +30,29 @@
 								<!-- IF users.usernameSpam -->
 								<i class="fa fa-times-circle text-danger" title="[[admin/manage/registration:list.username-spam, {users.spamData.username.frequency}, {users.spamData.username.appears}, {users.spamData.username.confidence}]]"></i>
 								<!-- ELSE -->
-								<i class="fa fa-check text-success"></i>
+                                    <!-- IF users.spamChecked -->
```
Comment (julianlam) [inline src/views/admin/manage/registration.tpl]: More indentation fixes here :smile:

File: src/views/admin/manage/registration.tpl [hunk 3]
```diff
@@ -50,9 +50,9 @@
 								<!-- IF users.ipSpam -->
 								<i class="fa fa-times-circle text-danger" title="[[admin/manage/registration:list.ip-spam, {users.spamData.ip.frequency}, {users.spamData.ip.appears}]]"></i>
 								<!-- ELSE -->
-								    <!-- IF users.spamChecked -->
-        							<i class="fa fa-check text-success"></i>
-    								<!-- ENDIF users.spamChecked -->
+                                    <!-- IF users.spamChecked -->
+                                    <i class="fa fa-check text-success"></i>
+                                    <!-- ENDIF users.spamChecked -->
```
Comment (akhoury) [inline src/views/admin/manage/registration.tpl]: @julianlam i am not sure why it's indented that way :/ it looks fine in my editor 

i'm gonna try something

screen shot 2019-01-25 at 3 59 38 pm

Comment (pitaj) [reply src/views/admin/manage/registration.tpl]: You must have a different tab width set up, and also you have spaces instead of tabs.

Comment (akhoury) [reply src/views/admin/manage/registration.tpl]: shouldn't the tab width be set by editorconfig https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#tab_width

Comment (akhoury) [reply src/views/admin/manage/registration.tpl]: I don't think i have spaces, my editor is following editorconfig

**Pull request closed without merging**